docker-compose.yml

```
@@ -0,0 +1,22 @@
+version: "3.8"
+services:
+  wordpress:
+    image: wordpress:latest
```

Sry, missed one thing, U should use create image, not just pull  image: wordpress:latest
```suggestion
    build:
      context: .
      dockerfile: Dockerfile
    container_name: wordpress
```

in context of this tasks it will lead to same result but if we imagine that someone will change Dockerfile ur docker-compose will still use just common wordpress image